flags/flags.go

```diff
@@ -0,0 +1,95 @@
+package flags
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/alecthomas/kong"
+	parcaflags "github.com/parca-dev/parca-agent/flags"
+	log "github.com/sirupsen/logrus"
```

Comment:
not important, but if this project grows bigger we should replace this with slog, which is slowly becoming the standard in the community (since it's being driven by the core Go team)

Reply:
yes, let's do that now.